packages/nimble-components/src/wafer-map/modules/computations.ts

```diff
@@ -249,13 +249,15 @@ export class Computations {
             .paddingOuter(0)
             .align(0)
             .round(false);
+        // html canvas has top-left origin https://www.w3schools.com/graphics/canvas_coordinates.asp
```

Comment:
nit/pref, prefer mdn docs

```suggestion
        // html canvas has top-left origin https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial/Drawing_shapes#the_grid
```